test/common/libraries/os_properties.rb

```diff
@@ -22,4 +22,8 @@ def redhat_ubi?
   def alinux2?
     inspec.os.name == 'amazon' && inspec.os.release.to_i == 2
   end
+
+  def ubuntu
+    inspec.os.name == 'ubuntu'
+  end
```

Comment:
Nitpick: right now it is the same as `os.debian?`, I would avoid adding unnecessary functions.
Also, please use final `?` for functions returning boolean values.